src/Parser/HolyPriest/CombatLogParser.js

```diff
@@ -0,0 +1,271 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import Icon from 'common/Icon';
+import ITEMS from 'common/ITEMS';
+
+import StatisticBox from 'Main/StatisticBox';
+import SuggestionsTab from 'Main/SuggestionsTab';
+import TalentsTab from 'Main/TalentsTab';
+import CastEfficiencyTab from 'Main/CastEfficiencyTab';
+import CooldownsTab from 'Main/CooldownsTab';
+import ManaTab from 'Main/ManaTab';
+
+import MainCombatLogParser from 'Parser/Core/CombatLogParser';
+import getCastEfficiency from 'Parser/Core/getCastEfficiency';
+import ISSUE_IMPORTANCE from 'Parser/Core/ISSUE_IMPORTANCE';
+
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
```

Comment:
You don't need to import this if you made no modifications; it's loaded automatically by the core `CombatLogParser`.